src/models/product.model.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+
+const { toJSON, paginate } = require('./plugins');
+
+const { Schema } = mongoose;
+
+const ProductSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String,
```

Comment:
Do we have another type called URL? or just String?

Reply:
We only have a String data type. However, if we need it, we can use [validator](https://www.npmjs.com/package/validator) to validate the data for the property to ensure it only contains the data specified.